src/components/strategies/marketPriceIndication/MarketPriceIndication.test.tsx

```diff
@@ -23,7 +15,7 @@ describe('MarketPriceIndication', () => {
     );
 
     const indicationText = screen.getByText('5.00% above market');
-    expect(indicationText).toBeTruthy();
+    expect(indicationText).toBeInTheDocument();
```

Comment:
this is probably a redundant assertion because if it wasn't present the previous line would throw:
https://testing-library.com/docs/queries/about#types-of-queries